components/post_view/commented_on/commented_on.tsx

```diff
@@ -17,7 +17,7 @@ import UserProfile from '../../user_profile/user_profile';
 type Props = {
     displayName?: string;
     enablePostUsernameOverride?: boolean;
-    parentPostUser?: UserProfileType;
+    parentPostUser?: UserProfileType | null;
```

Comment:
doesn't having the `?` operator already allows values to be `null`?

Reply:
No, it will assume the `parentPostUser` will only have `UserProfileType` and undefined not null